docs/assistant/cloud/libraries/view-edit-assets.md

```diff
@@ -0,0 +1,25 @@
+---
+id: view-edit-assets
+title: View & Edit Assets
+sidebar_label: View & Edit Assets
+---
+
+## View and edit assets in the Assistant Pro Cloud
+
+To view the attributes of or edit a creative asset from your Assistant Pro Cloud dashboard, click the asset to open a sidebar.  The sidebar will reveal the asset’s attributes and allow you to perform various actions including assigning library collections.  The item pictured here is assigned to two collections, Pastel Colors and Pink Colors.
+
+<img src="https://plchldr.co/i/800x300?&bg=f6f6f6&fc=656565&text=Placeholder" />
+
+## View and edit assets in the Assistant Plugin
+
+To view the attributes of and edit creative assets in the Assistant Plugin, click on the asset from your library.  Clicking the asset will open a window that will reveal the asset’s attributes and allow you to perform various actions including:
```

Comment:
will open a window that will reveal the asset's attributes and allow -> opens a window that displays the asset's attributes and allows

Reply:
Fixed this in my PR